scrape/manager_test.go

```diff
@@ -57,15 +57,17 @@ func init() {
 }
 
 func TestPopulateLabels(t *testing.T) {
+	t.Parallel()
```

Comment:
For tests that take less than `10ms`, no need to add parallelization I think to avoid the overhead. I see there are many other fast tests in here, I think we can skip them.